src/back/TicketManager/src/Core/TicketManager.Core.Services/Endpoints/Accounts/AccountLoginEndpoint.cs

```diff
@@ -34,6 +36,16 @@ public override async Task HandleAsync(AccountLoginRequest req, CancellationToke
             .FirstOrDefaultAsync(ct);
         if (account is not null && passwordManager.DoPasswordsMatch(account.PasswordHash, req.Password))
         {
+            if (account.Role == AccountRoles.Organizer)
+            {
+                var organizer = await coreDbContext.Organizers.Where(a => a.Id == account.Id).FirstOrDefaultAsync(ct);
+
+                if (!organizer?.IsVerified() ?? false)
```

Comment:
I'm not sure if `false` is correct here. It probably doesn't matter since the situation is impossible to have organizer's `Account` class without `Organizer`